webviz_config/_config_parser.py

```diff
@@ -299,81 +298,38 @@ def clean_configuration(self) -> None:
         if "title" not in self.configuration:
             self.configuration["title"] = "Webviz - Powered by Dash"
 
-        if "pages" not in self.configuration:
+        if "pages" not in self.configuration and "layout" not in self.configuration:
             raise ParserError(
                 f"{terminal_colors.RED}{terminal_colors.BOLD}"
                 "The configuration file does not have "
                 "information regarding which pages to create."
                 f"{terminal_colors.END}"
             )
 
-        if not isinstance(self.configuration["pages"], list):
+        if "pages" in self.configuration and not isinstance(
+            self.configuration["pages"], list
+        ):
```

Comment:
If `pages` is in the configuration file we decided that we should not emit a `UserWarning` for now (to give users time to move by themleves to the new features), but maybe we could give an indication to other developers (i.e. plugin projects) that `pages` should be replaced? I.e. add for example a `PendingDeprecationWarning`? (https://docs.python.org/3/library/warnings.html#warning-categories). These warnings are by default not shown.